lib/shopify-cli/commands/populate/resource.rb

```diff
@@ -90,57 +72,24 @@ def input_options
           end
         end
 
-        def input_fields
-          @input_fields = schema[self.class.input_type]['inputFields'].each_with_object({}) do |field, obj|
-            obj[field['name']] = field
-            obj
-          end
-        end
-
-        def to_input(struct)
-          struct.to_h.map do |key, value|
-            value = "\"#{value}\"" if input_fields[key.to_s]['type']['name'] == 'String'
-            "#{key}: #{value}"
-          end.join(',')
-        end
-
-        def mutation
-          <<~MUTATION
-            mutation {
-              #{self.class.field}(input: {#{to_input(@input)}}) {
-                #{self.class.type} {
-                  #{payload}
-                }
-              }
-            }
-          MUTATION
-        end
-
-        def payload
-          schema[Helpers::String.cap_first(self.class.type)]['fields']
```

Comment:
I think this is the only place I'm using this capitalization helper, so that can be removed.